packages/smooth_app/lib/pages/hunger_games/question_page.dart

```diff
@@ -492,3 +498,16 @@ class _ErrorLoadingView extends StatelessWidget {
     );
   }
 }
+
+/// Holds the number of questions answered
```

Comment:
Hopefully we don't hit edge cases where new questions are added on the fly, as the user answers the first questiona. And it will just be a missed opportunity.